src/lib/storage/disk_analyzer.rb

```diff
@@ -286,10 +286,14 @@ def installation_disk?(disk)
       # minor/major IDs since the inst-sys is in a RAM disk (copied from the
       # installation medium).
       #
-      def installation_volume?(vol)
-        log.info("Checking if #{vol.name} is an installation volume")
-        is_inst = mount_and_check(vol) { |mp| installation_volume_check(mp) }
-        log.info("#{vol} is installation medium") if is_inst
+      # @param vol_name [string] device name of the volume to check
+      #
+      # @return [bool] 'true' if the volume is an installation volume
+      #
+      def installation_volume?(vol_name)
+        log.info("Checking if #{vol_name} is an installation volume")
+        is_inst = mount_and_check(vol_name) { |mp| installation_volume_check(mp) }
+        log.info("#{vol_name} is installation medium") if is_inst
```

Comment:
There is a summary of all that later anyway:

2015-12-21 15:09:44 <1> morgul(26268) [Ruby] storage/disk_analyzer.rb:187 Linux part: ["/dev/sda5", "/dev/sda1", "/dev/sda3", "/dev/sda2"]
2015-12-21 15:09:44 <1> morgul(26268) [Ruby] storage/disk_analyzer.rb:95 Linux partitions found - not checking for Windows partitions
2015-12-21 15:09:44 <1> morgul(26268) [Ruby] storage/disk_analyzer.rb:98 Installation disks: []
2015-12-21 15:09:44 <1> morgul(26268) [Ruby] storage/disk_analyzer.rb:99 Candidate    disks: ["/dev/sda"]
2015-12-21 15:09:44 <1> morgul(26268) [Ruby] storage/disk_analyzer.rb:100 Linux   partitions: ["/dev/sda5", "/dev/sda1", "/dev/sda3", "/dev/sda2"]
2015-12-21 15:09:44 <1> morgul(26268) [Ruby] storage/disk_analyzer.rb:101 Windows partitions: []